src/client/src/orangehrmCorporateDirectoryPlugin/pages/CorporateDirectory.vue

```diff
@@ -0,0 +1,253 @@
+<!--
+/**
+ * OrangeHRM is a comprehensive Human Resource Management (HRM) System that captures
+ * all the essential functionalities required for any enterprise.
+ * Copyright (C) 2006 OrangeHRM Inc., http://www.orangehrm.com
+ *
+ * OrangeHRM is free software; you can redistribute it and/or modify it under the terms of
+ * the GNU General Public License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * OrangeHRM is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with this program;
+ * if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA  02110-1301, USA
+ */
+ -->
+
+<template>
+  <div class="orangehrm-background-container">
+    <oxd-table-filter :filter-title="$t('***.directory')">
+      <oxd-form>
+        <oxd-form-row>
+          <oxd-grid :cols="3" class="orangehrm-full-width-grid">
+            <oxd-grid-item>
+              <employee-autocomplete />
+            </oxd-grid-item>
+            <oxd-grid-item>
+              <jobtitle-dropdown />
+            </oxd-grid-item>
+            <oxd-grid-item>
+              <subunit-dropdown />
+            </oxd-grid-item>
+          </oxd-grid>
+        </oxd-form-row>
+
+        <oxd-divider />
+
+        <oxd-form-actions>
+          <oxd-button
+            :label="$t('general.reset')"
+            display-type="ghost"
+            type="reset"
+          />
+          <oxd-button
+            :label="$t('general.search')"
+            class="orangehrm-left-space"
+            display-type="secondary"
+            type="submit"
+          />
+        </oxd-form-actions>
+      </oxd-form>
+    </oxd-table-filter>
+    <br />
+    <div :class="!isTotalZero ? 'orangehrm-corporate-directory' : ''">
```

Comment:
refactor this as @Super-Chama mentioned above
```suggestion
    <div :class="{'orangehrm-corporate-directory': !isTotalZero}">
```

Reply:
Done